docs/content/preview/manage/backup-restore/lightweight-db-clone.md

```diff
@@ -0,0 +1,258 @@
+CREATE DATABASE clone_db TEMPLATE original_db;
+```
+
+In this example, `clone_db` is created as a clone of `original_db`, and contains the latest schema and data of `original_db` as of current time.
+
+To create a clone of the original database at a specific point in time (within the history retention period specified when creating the snapshot schedule), you can specify the [Unix timestamp](https://www.unixtimestamp.com/) in microseconds using the `AS OF` option as follows:
+
+```sql
+CREATE DATABASE clone_db TEMPLATE original_db AS OF 1723146703674480;
+```
+
+### Clone a YCQL keyspace
+
+You can create a clone in YCQL using the yb-admin `clone_namespace` command as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS clone_namespace ycql.originaldb1 clonedb2 1715275616599020
+```
+
+In this example, `clonedb2` is created as a clone of `originaldb1` as of 1715275616599020 Unix timestamp.
+
+### Check the clone status
+
+To check the status of clone operations performed on a database, use the yb-admin `list_clones` command and provide the `source_database_id` (YSQL) or `source_namespace_id` (YCQL), as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000
+```
+
+```output
+[
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "1",
+        "target_namespace_name": "testing_clone_db",
+        "restore_time": "2024-08-09 21:42:16.451974"
+    },
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "2",
+        "target_namespace_name": "dev_clone_db",
+        "restore_time": "2024-08-09 21:42:55.048663"
+    }
+]
+```
+
+You can find the `source_database_id` or `source_namespace_id` from the [YB-Master leader UI](../../../reference/configuration/default-ports/#servers) under the `/namespaces` end point.
+
+In this example, two clones were made of the source database `00004000000030008000000000000000` that are COMPLETE. The two clones are `testing_clone_db` and `dev_clone_db` and they each have a unique `seq_no` used to identify each clone operation from the same source database.
+
+You can check the status of a specific clone operation if you have both the `source_database_id` (YSQL) or `source_namespace_id`(YCQL) and the `seq_no` as follows:
+
+```sh
+./bin/yb-admin --master_addresses $MASTERS list_clones 00004000000030008000000000000000 2
+```
+
+```output
+[
+    {
+        "aggregate_state": "COMPLETE",
+        "source_namespace_id": "00004000000030008000000000000000",
+        "seq_no": "2",
+        "target_namespace_name": "dev_clone_db",
+        "restore_time": "2024-08-09 21:42:55.048663"
+    }
+]
+```
+
+Use the `list_clones` command to check whether a clone operation completed successfully or not.
+
+Note that the cluster doesn't allow two clone operations to happen concurrently on the same source database.
```

Comment:
giving this another look, I think we should make it clearer as it caused some confusion internally so we should be clearer with our public-facing docs. In this regard, I suggest:
```suggestion
Note that the cluster doesn't allow you to perform two clone operations concurrently on the same source database. You have to wait for the first clone to finish until you can perform another clone. However, you can create as many clones as you want for the same database or different databases.
```